cypress/support/fragments/finance/funds/fundDetails.js

```diff
@@ -1,20 +1,40 @@
 import {
+  Button,
   KeyValue,
   MultiColumnListCell,
   MultiColumnListRow,
+  PaneHeader,
   Section,
   including,
 } from '../../../../../interactors';
 import BudgetDetails from '../budgets/budgetDetails';
+import Transactions from '../transactions/transactions';
 
 const fundDetailsPane = Section({ id: 'pane-fund-details' });
+
+// fund details header
+const fundDetailsPaneHeader = PaneHeader({ id: 'paneHeaderpane-fund-details' });
+const actionsButton = Button('Actions');
+
 const currentBudgetSection = Section({ id: 'currentBudget' });
 const previousBudgetsSection = Section({ id: 'previousBudgets' });
 
 export default {
   waitLoading: () => {
     cy.expect(fundDetailsPane.exists());
   },
+  expandActionsDropdown() {
+    cy.do(fundDetailsPaneHeader.find(actionsButton).click());
+  },
+  viewTransactionsForCurrentBudget() {
+    this.expandActionsDropdown();
+
+    cy.do(Button('View transactions for current budget').click());
+
+    Transactions.waitLoading();
+
+    return Transactions;
```

Comment:
probably need to use common approach without returning page

Reply:
From my point of view this approach makes it more obvious what fragment can be used on each next step and this reduces duplication of same functionality in different modules.